src/main.rs

```diff
@@ -88,6 +88,13 @@ enum Opts {
         )]
         hidden: bool,
 
+        #[structopt(
+            short = "g",
```

Comment:
probably less missleading

```suggestion
            short = "G",
```